build.gradle

```diff
@@ -39,22 +45,26 @@ ext {
   slf4jVersion = '1.7.36'
   vavrVersion = '0.10.4'
 
```

Comment:
Btw I'm now re-checking the part about Java 11 check... isn't it just enough simply ditch the `task` and `afterEvaluate` part, and just put

```
if(JavaVersion.current() != JavaVersion.VERSION_11) {
  throw new GradleException("Project must be built with version 11")
}
```

directly as top-level instructions in build.gradle? 🤔

Reply:
yess, from what I checked, we have the same result